### PR TITLE
0.1.1 Release

### PR DESCRIPTION
**Pull request by tanjpg**
This Pull Request adds in the 0.1.1 release of the `easy_manipulation_deployment` package

### CODE REVIEW

File: docs/sphinx/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'EMD'
+copyright = '2020, Glenn Tan'
```
Comment (AlanXue-ROSI) [inline docs/sphinx/source/conf.py:21]: 2021 ROS-Industrial?

File: easy_manipulation_deployment/grasp_planner/include/grasp_planner/common/math_functions.hpp [hunk 2]
```diff
@@ -24,8 +24,30 @@
...
+  float perpendicular_distance;
+  int index;
+  pcl::PointNormal pcl_npoint;
+  Point(
+    Eigen::Vector3f reference_vector_,
+    Eigen::Vector3f reference_point_,
+    float projection_distance_,
+    float perpendicular_distance_)
+  {
+    reference_vector = reference_vector_;
```
Comment (AlanXue-ROSI) [inline easy_manipulation_deployment/grasp_planner/include/grasp_planner/common/math_functions.hpp:45]: use initializer list

The reason is https://stackoverflow.com/questions/9903248/initializing-fields-in-constructor-initializer-list-vs-constructor-body


```
#include <iostream>

using namespace std;

struct TestStruct {
        int id;
        TestStruct() : id(42)
        {
        }
};

int main()
{
    TestStruct t;
    cout<<"Hello World" << t.id;

    return 0;
}

```

**Review by AlanXue-ROSI — COMMENTED** [review]
A quick scan. 
The review is done.

File: docs/sphinx/source/conf.py (hunk 1 above)
Comment (tanjpg) [reply docs/sphinx/source/conf.py]: Reverted commit to add docs folder back into the repo as we have hosted the docs separately. This file will no longer be present

File: easy_manipulation_deployment/grasp_planner/include/grasp_planner/common/math_functions.hpp (hunk 2 above)
Comment (tanjpg) [reply easy_manipulation_deployment/grasp_planner/include/grasp_planner/common/math_functions.hpp]: Rectified. 75d3c9a

**Review by cardboardcode — CHANGES_REQUESTED** [review]
Pull Request ready to merge. 

Please change the `emd_logo.png` to `emd_logo_long.png` in `README.md`. 
**Rationale**: This will help make the `README.md` formatting more coherent.

The `emd_logo_long.png` will be provided internally via private communication channels.

**Review by cardboardcode — APPROVED** [review]

**Pull request merged**